.github/workflows/connect-release-init.yml

```diff
@@ -0,0 +1,46 @@
+name: Connect v9 release init
```

Comment:
what about starting some namin convention for the workflow names ? `release - ` `test - ` and so on. This could then be `release - connect v9` or something like that if you have any idea.